src/lettercrap.ts

```diff
@@ -1,3 +1,8 @@
+/*!
+ * @licence MIT (https://lettercrap.github.io/lettercrap/LICENSE)
+ * @author Ozren Dabić
+ * @author Davide Ciulla
+ */
 import { z } from 'zod';
```

Comment:
I would maybe add a blank file before the zod import because the license statement looks like it's linked to the import